docs/source/quickstart.md

```diff
@@ -0,0 +1,37 @@
+(quickstart)=
+## Quickstart
+Robbie now uses Nextflow to manage the workflow and can be run on a local system or a supercomputing cluster. You can use a container via singularity, docker, or the host's software. The current development cycle tests Robbie using singularity on an HPC with the Slurm executor - other setups *should* work but haven't been extensively tested.
+
+### `images.txt`
+Before running Robbie, you will need to create a text file that contains the paths to each image to be processed. For example, if within the "Robbie" parent directory there exists a folder named "images" containing the `.fits` files:
+
+``` bash
+ls images/* > images.txt
+```
+
+will populate `images.txt` with the image paths relative to the parent directory.
+
+### `robbie.nf`
+This file describes the workflow and can be inspected but shouldn't be edited directly. To describe the command line arguments, use
+``` bash
+robbie.nf --help
+```
+
+### `nextflow.config`
+This file is the configuration setup and contains all the command line arguments' default values. You can change these defaults by copying the `nextflow.config` and editing the relevant params.\<argument\>. You can then use your custom config via:
+``` bash
+nextflow -C my.config run robbie.nf
+```
+The `-C my.config` directs Nextflow to use *only* the configuration described in `my.config`. If you use `-c`, then it will also read the `nextflow.config` file.
```

Comment:
I think the second sentence is clearer as:
If you use `-c`, you can supply a partial config file that will only overwrite the defaults in the partial config file.

Reply:
I'm not super clear on the behaviour of Nextflow here. If I specify `-c my.config` does it only overwrite the defaults if they're specified in that config file? Where are the defaults stored?

Reply:
`-c my.config` will look for an use `nextflow.config` if it exists, then load `my.config`. It is possible to define things in the `my.config` file that are not in the `nextflow.config` file but it's probably not a good idea.

`-C my.config` will only use that one file for reading configuration. No other files will be used.

Reply:
Thanks for the clarification. So maybe I should say something like:

> If you use ``-c``, you can supply a config file that is supplementary to ``nextflow.config``, for example ``my.config``. If both config files have the same parameters defined, ``my.config`` takes precedence otherwise the parameters are merged.

Reply:
Perfect